react-components/jsx.md

```diff
@@ -12,34 +12,26 @@ Before we start using JSX extensively, let's pause to understand what JSX is and
 
 ## Introduction
 
-In a newly created React app using `create-react-app`, we can see the following code snippets:
+In a newly created React app using Vite, we can see the following code snippets:
 
-`src/index.js`
+`src/main.jsx`
 
 ```js
 <React.StrictMode>
   <App />
-</React.StrictMode>
+</React.StrictMode>,
 ```
 
-`src/App.js`
+`src/App.jsx`
 
 ```js
-<div className="App">
-  <header className="App-header">
-    <img src={logo} className="App-logo" alt="logo" />
-    <p>
-      Edit <code>src/App.js</code> and save to reload.
-    </p>
-    <a
-      className="App-link"
-      href="https://reactjs.org"
-      target="_blank"
-      rel="noopener noreferrer"
-    >
-      Learn React
-    </a>
-  </header>
+<div className="card">
+  <button onClick={() => setCount((count) => count + 1)}>
+    count is {count}
+  </button>
+  <p>
+    Edit <code>src/App.jsx</code> and save to test HMR
+  </p>
 </div>
 ```
 
```

Comment:
How about adding something like: "Note that the tilde is part of the message and **not** special syntax." when `Welcome~` first appears in the lesson?

Reply:
If we're going to make a change regarding this at all, I'd really prefer to completely remove the tilde. It's just so unnecessary.

Reply:
I'm ok to remove it since it's so minor, but when students generate their own project it'll show up on their end. Hopefully it'll just go unnoticed by students?

Reply:
Isn't the tilde just in sample example code that we show? It's not in the vite template that I see.

Reply:
OMG sorry, I was conflating what was already in Learn with new Vite stuff, you're right. I'll remove the tilde